tests/unit/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenThread Unit Tests
+
+This page shows how to build and run OpenThread unit tests. It will be helpful for developers to debug failed unit test cases if they got one in CI or to add some new test cases.
+
+## Build Simulation
```

Comment:
This is useful. Thanks @Irving-cl .

I usually use this one command `./tests/toranj/build.sh cmake`  
- This sets up and builds everything (cli/ncp ftd/mtd/rcp) and all the unit tests. 
- Afterwards can use `ninja` from the openthread root folder to re-build (if there is any change).
- What I like about this is that unit test executable will be in the same folder as the source files (can run like `./tests/unit/ot-test-message`)